src/js/views/patients/worklist/worklist_views.js

```diff
@@ -213,6 +214,20 @@ const ListView = CollectionView.extend({
       });
     });
   },
+  onClickShiftMultiSelect(selectedActionId) {
+    const actionsList = this.children._views;
+    const lastSelectedActionId = this.state.getLastSelected();
+
+    const selectedIndex = actionsList.findIndex(action => action.model.id === selectedActionId);
+    const lastSelectedIndex = actionsList.findIndex(action => action.model.id === lastSelectedActionId);
+
+    const arrayOfActionIdsToSelect = actionsList
+      .slice(Math.min(selectedIndex, lastSelectedIndex), Math.max(selectedIndex, lastSelectedIndex) + 1)
+      .map(action => action.model.id);
+
+    this.state.selectMultiple(arrayOfActionIdsToSelect);
```

Comment:
Well now that I think we should rename `selectAll` we need to pass a collection here.  that'd work like the `this.filteredCollection`.  In fact we could utilize this collection as it is a model dump of the views filtered by find in list.

I think it'd be something like:
```js
const selectedCollection = this.filteredCollection.clone();
selectedCollection.reset(selectedCollection.filter(model => {
  const view = this.children.findByModel(model);
  const index = this.children.findIndexByView(view);
  return  (index > minIndex) && (index < maxIndex);
}));
this.state.selectMultiple(selectedCollection);
```